web/models.py

```
@@ -50,18 +51,20 @@ def __str__(self):
     # Avatar fields
     avatar = models.ImageField(upload_to="avatars", blank=True, default="")
     custom_avatar = models.OneToOneField(
         "Avatar", on_delete=models.SET_NULL, null=True, blank=True, related_name="profile"
     )
     is_teacher = models.BooleanField(default=False)
     is_social_media_manager = models.BooleanField(default=False)
-    discord_username = models.CharField(max_length=50, blank=True, help_text="Your Discord username (e.g., User#1234)")
-    slack_username = models.CharField(max_length=50, blank=True, help_text="Your Slack username")
-    github_username = models.CharField(max_length=50, blank=True, help_text="Your GitHub username (without @)")
+    discord_username = EncryptedCharField(
+        max_length=50, blank=True, help_text="Your Discord username (e.g., User#1234)"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)